pulsar-common/src/main/java/org/apache/pulsar/common/util/keystoretls/KeyStoreSSLContext.java

```diff
@@ -339,7 +340,10 @@ public static SSLContext createClientSslContext(String keyStoreTypeString,
         return keyStoreSSLContext.createSSLContext();
     }
 
-    // for web server. autoRefresh is default true.
+    /**
+     * for web server. autoRefresh is default true.
+     * @deprecated Use JettySslContextFactory.createServerSslContextWithKeystore instead.
+     */
     public static SslContextFactory.Server createSslContextFactory(String sslProviderString,
                                                                    String keyStoreTypeString,
                                                                    String keyStore,
```

Comment:
Please remove this method completely. It was there to be used in Jetty.

Drop references to the Jetty support since pulsar-common is also used on the client. no need to have a deprecation cycle since this is not a public API for Pulsar end users.

Reply:
Fixed.